options.mk

```
@@ -74,7 +74,7 @@ credentials+discovery+ecommerce+edx_notes_api+forum+frontend-app-publisher+front
 # Separated by plus signs.
 # Separated by plus signs. Listed in alphabetical order for clarity.
 EDX_SERVICES ?= \
-analyticspipeline+credentials+discovery+ecommerce+edx_notes_api+forum+frontend-app-learning+frontend-app-publisher+gradebook+lms+lms_watcher+marketing+program-console+registrar+registrar-worker+studio+studio_watcher+xqueue+xqueue_consumer
+analyticspipeline+credentials+discovery+ecommerce+edx_notes_api+forum+frontend-app-learning+frontend-app-publisher+frontend-app-library-authoring+gradebook+lms+lms_watcher+marketing+program-console+registrar+registrar-worker+studio+studio_watcher+xqueue+xqueue_consumer
```

(the tiniest of nits -- an alphabetization fix)
```suggestion
analyticspipeline+credentials+discovery+ecommerce+edx_notes_api+forum+frontend-app-learning+frontend-app-library-authoring+frontend-app-publisher+gradebook+lms+lms_watcher+marketing+program-console+registrar+registrar-worker+studio+studio_watcher+xqueue+xqueue_consumer
```

---

Done!